reloadButton.js

```diff
@@ -0,0 +1,19 @@
+var href = window.location.href
```

Comment:
we must not have a test that runs the hyperdom mount in node - this would fail
you need to do `var window = require('global')`